### ISSUE TITLE
[YSQL][LST] ERROR: Already present: Duplicate request 14 from client ... (min running 14)

### ISSUE THREAD

**Issue by def-** [yugabyte/yugabyte-db#14368]
Jira Link: [DB-3791](https://yugabyte.atlassian.net/browse/DB-3791)
### Description

```
$ cd ~/code/yugabyte-db
$ git checkout 14a71540ec1634c926c4865c002e945088d119d2
$ ./yb_build.sh release
$ bin/yb-ctl --replication_factor 3 create --tserver_flags=enable_deadlock_detection=true,ysql_max_connections=20,ysql_enable_packed_row=true,yb_enable_read_committed_isolation=true,ysql_num_shards_per_tserver=2,enable_stream_compression=true,stream_compression_algo=2,yb_num_shards_per_tserver=2 --master_flags=yb_enable_read_committed_isolation=true,enable_stream_compression=true,stream_compression_algo=2,enable_automatic_tablet_splitting=true,tablet_split_low_phase_shard_count_per_node=1,tablet_split_high_phase_shard_count_per_node=5,ysql_enable_packed_row=true,enable_deadlock_detection=true
$ cd ~/code/yb-long-system-test
$ git checkout f76f4c65
$ ./long_system_test.py --nodes=127.0.0.1:5433,127.0.0.2:5433,127.0.0.3:5433 --threads=10 --runtime=60 --max-columns=20 --complexity=full --seed=076005
2022-09-29 13:11:49,322 MainThread INFO     Reproduce with: git checkout f76f4c65 && ./long_system_test.py --nodes=127.0.0.1:5433,127.0.0.2:5433,127.0.0.3:5433 --threads=10 --runtime=60 --max-columns=20 --complexity=full --seed=076005
2022-09-29 13:11:50,051 MainThread INFO     Database version: PostgreSQL 11.2-YB-2.15.4.0-b0 on x86_64-pc-linux-gnu, compiled by clang version 14.0.6 (https://github.com/yugabyte/llvm-project.git 4047555d02fa41c8dbfc8a8f529680276af9742e), 64-bit
2022-09-29 13:11:50,054 MainThread INFO     Creating tables for database db_lst_076005
2022-09-29 13:12:25,226 MainThread INFO     Starting worker_0: CreateIndexAction, DropIndexAction, SetConfigAction, AddColumnAction
2022-09-29 13:12:25,229 MainThread INFO     Starting worker_1: RandomSelectAction, SetConfigAction
2022-09-29 13:12:25,240 MainThread INFO     Starting worker_2: SingleInsertAction, SingleUpdateAction, SingleDeleteAction, BulkInsertAction, BulkUpdateAction, SetConfigAction
2022-09-29 13:12:25,260 MainThread INFO     Starting worker_3: RandomSelectAction, SetConfigAction
2022-09-29 13:12:25,274 MainThread INFO     Starting worker_4: RandomSelectAction, SetConfigAction
2022-09-29 13:12:25,279 MainThread INFO     Starting worker_5: SingleInsertAction, SingleUpdateAction, SingleDeleteAction, BulkInsertAction, BulkUpdateAction, SetConfigAction
2022-09-29 13:12:25,285 MainThread INFO     Starting worker_6: SingleInsertAction, SingleUpdateAction, SingleDeleteAction, BulkInsertAction, BulkUpdateAction, SetConfigAction
2022-09-29 13:12:25,286 MainThread INFO     Starting worker_7: RandomSelectAction, SetConfigAction
2022-09-29 13:12:25,334 MainThread INFO     Starting worker_8: SingleInsertAction, SingleUpdateAction, SingleDeleteAction, BulkInsertAction, BulkUpdateAction, SetConfigAction
2022-09-29 13:12:25,475 MainThread INFO     Starting worker_9: RandomSelectAction, SetConfigAction
2022-09-29 13:12:35,499 MainThread INFO     Worker queries/s: [000.0][000.9][001.1][000.8][000.7][002.5][000.7][000.3][001.1][000.4]
2022-09-29 13:12:45,518 MainThread INFO     Worker queries/s: [000.0][000.3][000.1][000.6][000.2][000.4][000.4][000.4][000.0][000.1]
2022-09-29 13:12:55,544 MainThread INFO     Worker queries/s: [000.0][000.0][000.0][000.2][000.0][000.0][000.0][000.0][000.0][000.0]
2022-09-29 13:12:59,903 worker_2   ERROR    Unexpected query failure: InternalError_
Query: INSERT INTO tg2_0 (c0_date, c1_float4, c2_int4range, c3_smallint, c4_int8range, c5_float8, c6_json, c7_int, c8_int4range) VALUES ('1987-08-18', 58.10929428007245, '(80,85)'::INT4RANGE, 75, '(-89,37)'::INT8RANGE, 10.052095581830315, '{"a": 5, "b": ["0", "1", "2", "3", "4", "5", "6", "7"], "c": false}'::json, -30, '[-75,84]'::INT4RANGE);
  values: None
  runtime: 2022-09-29 13:12:36.090 - 2022-09-29 13:12:59.893
  supports explain: True
  supports rollback: True
  affected rows: None
Action: SingleInsertAction
Error class: InternalError_
Error code: XX000
Error message: ERROR:  Already present: Duplicate request 14 from client 1666c86e-d225-4fa2-8755-bac12785aa11 (min running 14)
    @           0x99bb28  errmsg
    @           0x9c6bf5  HandleYBStatusAtErrorLevel
    @           0x6bb30b  standard_ExecutorFinish
    @     0x7f9cec43bbb1  pgss_ExecutorFinish
    @     0x7f9cec4327aa  ybpgm_ExecutorFinish
    @           0x857f67  ProcessQuery
    @           0x857455  PortalRunMulti
    @           0x856cd6  PortalRun
    @           0x854456  yb_exec_simple_query_impl
    @           0x8549f6  yb_exec_query_wrapper_one_attempt
    @           0x85152b  PostgresMain
    @           0x7c04cc  BackendRun
    @           0x7bf913  ServerLoop
    @           0x7bbdbb  PostmasterMain
    @           0x71b13d  PostgresServerProcessMain
    @           0x71b5e2  main
    @     0x7f9cf0d07825  __libc_start_main
    @           0x4c90d9  _start

    @           0x99bb28  errmsg
    @           0x852deb  YBPrepareCacheRefreshIfNeeded
    @           0x850e7d  PostgresMain
    @           0x7c04cc  BackendRun
    @           0x7bf913  ServerLoop
    @           0x7bbdbb  PostmasterMain
    @           0x71b13d  PostgresServerProcessMain
    @           0x71b5e2  main
    @     0x7f9cf0d07825  __libc_start_main
    @           0x4c90d9  _start

CONTEXT:  Catalog Version Mismatch: A DDL occurred while processing this query. Try again.
Transaction isolation level: read uncommitted
DB Node: host: 127.0.0.3, port: 5433
DB Backend PID: 3172382
```
LST logs: [lst.zip](https://github.com/yugabyte/yugabyte-db/files/9731701/lst.zip)
Another query error I only noticed now while checking the failed runs, even though it happened a week ago.
I noticed there was a bug from 3 years ago about this failure, not sure if this should be considered a duplicate: https://github.com/yugabyte/yugabyte-db/issues/1208

**Comment by def-** [comment]
Also occurs sporadically in 2.8.10.0-b21 in itest-system:
```
testysqlddlsintxnblk-k8s-rf3: Start
	(     0.199s) User Login : Success
	(     0.034s) Refresh YB Version : Success
	(     0.046s) Setup Provider : Success
	(     0.012s) Updating Health Check Interval to 60000 sec : Success
	(   120.267s) Create universe jenk-isd195-e0d0404a70-20221024-212629 : Success
	(   122.532s) Table with manual Splitting : Success
	(   439.217s) With Normal table : Success
	(    21.437s) With colocated table : Success
	(   212.084s) Table with manual splitting and alter table clause : >>> Integration Test Failed <<< 
Already present: [Already present (yb/consensus/retryable_requests.cc:242): Duplicate request 1661 from client f3a5c972-881a-469a-8eee-766fc9d8b6b9 (min running 1661)]

	(     4.130s) Saved server log files and keys at /share/jenkins/workspace/itest-system-developer-gke2/logs/2.8.10.0_testysqlddlsintxnblk-k8s-rf3_20221024_222821 : Success
	(   180.247s) Destroy universe : Success
testysqlddlsintxnblk-k8s-rf3: End

```

**Comment by bmatican** [comment]
Hey @Huqicheng , passing this to you in case it's related to some of the tablet splitting work you've done recently, ie: if that would actually address this (since it was showing up on 2.8.1 with manual splits).

@def- meta note, we shouldn't try even manual splitting, on anything before 2.14, as tablet splitting as a whole was GAed then and we're not actively backporting relevant commits to 2.12 or below, for it, but only 2.14 and above.

**Comment by rthallamko3** [comment]
We believe this category of issues is fixed by #15006
